automation/jenkins_build.sh

```diff
@@ -0,0 +1,114 @@
+#!/bin/bash
+
+# comparing version: http://stackoverflow.com/questions/16989598/bash-comparing-version-numbers
+function version_ge() { test "$(echo "$@" | tr " " "\n" | sort -V | tail -n 1)" == "$1"; }
+
+# extract checksum for node binary
+function extract_checksum()
+{
+	# $1: binary type, 0: in-house, 1: official.
+	# $2: node version
+	# $3: variable name for result
+
+	local __resultVar=$3
+
+	if [ $1 -eq 0 ]; then
+		local __checksum=$(grep " node-v$2-linux-$binaryArch.tar.gz" SHASUMS256.txt)
+	else
+		curl -SLO "https://nodejs.org/dist/v$2/SHASUMS256.txt.asc" \
+		&& gpg --verify SHASUMS256.txt.asc \
+		&& local __checksum=$(grep " node-v$2-linux-$binaryArch.tar.gz\$" SHASUMS256.txt.asc) \
+		&& rm -f SHASUMS256.txt.asc
+	fi
+	eval $__resultVar="'$__checksum'"
+}
+
+# gpg keys listed at https://github.com/nodejs/node
+for key in \
+9554F04D7259F04124DE6B476D5A82AC7E37093B \
+94AE36675C464D64BAFA68DD7434390BDBE9B9C5 \
+0034A06D9D9B0064CE8ADF6BF1747F4AD2306D93 \
+FD3A5288F042B6850C66B31F09FE44734EB7990E \
+71DCFD284A79C3B38668286BC97EC7A07EDE3FC1 \
+DD8F2338BAE7501E3DD5AC78C273792F7D83545D \
+B9AE9905FFD7803F25714661B63B535A4C206CA9 \
+C4F0DFFF4E8C1A8236409D08E73BC641CC11F4C8 \
+7937DFD2AB06298B2293C3187D33FF9D0246406D \
+114F43EE0176B71C7BC219DD50A3051F888C628D \
+56730D5401028683275BD23C23EFEFE93C4CFFFE \
+; do \
+	gpg --keyserver ha.pool.sks-keyservers.net --recv-keys "$key"; \
+done
+
+resinUrl="http://resin-packages.s3.amazonaws.com/node/v\$NODE_VERSION/node-v\$NODE_VERSION-linux-#{TARGET_ARCH}.tar.gz"
+nodejsUrl="http://nodejs.org/dist/v\$NODE_VERSION/node-v\$NODE_VERSION-linux-#{TARGET_ARCH}.tar.gz"
+dateStamp=$(date +'%Y%m%d' -u)
+baseVersion=$(expr match "$NODE_VERSION" '\([0-9]*\.[0-9]*\)')
+
+for ARCH in $ARCHS
+do
+	case "$ARCH" in
+		'rpi')
+			baseImage='resin/rpi-raspbian:jessie'
+			binaryArch='armv6l'
+			binaryUrl=$nodejsUrl
+			extract_checksum 1 $NODE_VERSION "checksum"
+		;;
+		'armv7hf')
+			baseImage='resin/armv7hf-debian:jessie'
+			binaryArch='armv7hf'
+			binaryUrl=$resinUrl
+			extract_checksum 0 $NODE_VERSION "checksum"
+		;;
+		'armel')
+			baseImage='resin/armel-node:6.5-slim'
```

Comment:
@petrosagg please this issue https://github.com/resin-io-library/base-images/issues/313. I've updated the PR to make the script only builds armel node v0.10.x image.